src/cmd/connect.go

```diff
@@ -61,14 +62,12 @@ var connectCmd = &cobra.Command{
 
 		defer tunnel.Close()
 
-		// Dump the tunnel URL to the console for other tools to use.
-		fmt.Print(tunnel.FullURL())
```

Comment:
Wayne originally added this line, I asked him about it. Doesn't seem it was added for a specific use case. From what I can tell it's not even possible to pipe in, since the `zarf connect` command requires ctrl-c to exit. I feel comfortable removing.